src/main/java/edu/greenblitz/robotName/subsystems/swerve/modules/mk4iSwerveModule/MK4iSwerveConstants.java

```diff
@@ -47,24 +48,22 @@ public class MK4iSwerveConstants {
 		PID_CONFIGS.kS = ks;
 		PID_CONFIGS.kA = ka;
 		PID_CONFIGS.kV = kv;
-		PID_CONFIGS.kP = 40;
+		PID_CONFIGS.kP = 20;
 		PID_CONFIGS.kI = 0;
-		PID_CONFIGS.kD = 0;
+		PID_CONFIGS.kD = 1;
```

Comment:
Kd of one is pretty weird

Reply:
and? this is what we calibrated

Reply:
It’s just weird that you had to use kd and not max acceleration